README.md

```diff
@@ -145,6 +169,120 @@ such as Bloc and Provider
+- **Dialog**: Used for single user actions or to convey information related to changes in state (success, errors).
+- **Side Sheet**: Used to focus users' attention on a specific task while keeping the context visible.
+- **Bottom Sheet**: Used for additional options or actions.
+- **Full Bottom Sheet**: Used to present content that requires full screen height.
+
+### Safe Areas
+The Modal Sheet supports safe areas for all compatible devices. See below the safe area considerations.
+
+#### iOS
+
+1. **iPhone without Notch**:
+    - Status Bar Height: 20 points
+    - Safe Area Insets:
+        - Top: 20 points
+        - Bottom: 0 points (unless there's a home indicator)
+
+2. **iPhone with Notch (iPhone X and later)**:
+    - Status Bar Height: 44 points (varies with device)
+    - Safe Area Insets:
+        - Top: 44 points
+        - Bottom: 34 points (home indicator area)
+
+#### Android
+
+1. **Standard Devices**:
+    - Status Bar Height: Approximately 24 dp (varies with screen density and device)
+    - Safe Area Insets:
+        - Top: 24 dp (status bar)
+        - Bottom: Varies (on-screen navigation bars, if any)
+
+2. **Devices with Notch (Display Cutout)**:
+    - Status Bar Height: Varies depending on the notch size
+    - Safe Area Insets:
+        - Top: Varies with notch (retrieved using DisplayCutout API)
+        - Bottom: Varies (on-screen navigation bars, if any)
+
+### Scroll Logic
+
+The modal sheet component supports different scrolling behaviors to ensure usability and accessibility of the content:
+
+- **Non-scrollable Modals**: For content that fits within the viewport, no scrolling is required. The entire modal content is visible without interaction.
+- **Scrollable Modals**: When content exceeds the viewport height, the modal becomes scrollable. Users can scroll through the content while the modal's header and action buttons remain fixed for easy access.
+
+#### Scroll Logic Layouts
+
+- **Short Content**: Displayed fully within the viewport without scrolling.
+- **Medium Content**: Scrolls vertically if the content exceeds the available height but keeps action buttons fixed at the bottom.
+- **Long Content**: Utilizes full vertical scrolling. The header remains sticky at the top while action buttons are sticky at the bottom for continuous access.
+
+### Modal Sheet Layouts
+
+#### Alert
+- Used to display information requiring immediate user attention.
+- Adapts to different breakpoints to ensure visibility and accessibility.
+- Must be dismissed by user interaction to ensure the alert is acknowledged.
+
+#### Dialog
+- Used for single user actions or state change information (success, errors).
+- Scales with breakpoints to maintain usability.
+- Provides clear actions for users to acknowledge or dismiss the dialog.
+
+#### Side Sheet
+- Used to focus users' attention on specific tasks while keeping context visible.
+- Spans the full height of the viewport.
+- Is modal and blocks other interactions until dismissed.
```

Comment:
Is it worth mentioning this? because the package is modal sheet package and all modal types blocks other interactions until dismissed.

Reply:
removed